### PR TITLE
docs(lite-components): lite components limitations

### PR DESCRIPTION
**Pull request by voluntadpear**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

This PR mentions some of the limitations present with lite components.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by shairez** [comment]
thanks again @voluntadpear ! :)

File: packages/docs/src/routes/docs/components/lite-components/index.mdx [hunk 1]
```diff
@@ -34,3 +35,6 @@ In the above example, the `MyButton` is a lite component. Lite components' lazy
 - `MyButton` does not have a host element.
 
 You can think of lightweight components as being inlined into the component, where they are instantiated.
+
+> **Limitations**
+> Unlike regular Qwik components, lite components can not use `use` methods, such as `useSignal` or `useStore`. As it name implies, lite components are better used sparkly for lightweight pieces of markup since they offer the convenience of being bundled with the parent component.
```
Comment (tuurbo) [inline packages/docs/src/routes/docs/components/lite-components/index.mdx:40]: did you mean sparsely, not sparkly?

Comment (tuurbo) [reply packages/docs/src/routes/docs/components/lite-components/index.mdx:40]: https://github.com/BuilderIO/qwik/pull/2127

Comment (shairez) [reply packages/docs/src/routes/docs/components/lite-components/index.mdx:40]: merged, thanks @tuurbo !

**Pull request merged**